README.md

```diff
@@ -292,6 +292,36 @@ $builder = UnleashBuilder::create()
     ]);
 ```
 
+
+## Proxy SDK
+
+By default the SDK uses the Server-side endpoints on the Unleash API. You can also use the Proxy SDK, which is a
+lightweight SDK that uses the Client-side endpoints on the Unleash API. The Proxy SDK give a substantial performance improvement when using a large set of feature toggles (10K+).
+
+To use the Proxy SDK, you need to call `withProxy($apiKey)` on the builder. The `$apiKey` needs to be a [frontend token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens). Note that `withProxy($apiKey)` is in lieu of setting the API key header.

```

Comment:
sholdn't $apiKey be $frontendToken here then?

Reply:
I don't think so, no. This SDK (like most of our SDKs) calls this an API key (it does for the Server side token as well), I'm not all that keen to change the domain language here and especially not for users that are already using an API key